app/src/main/java/org/mifos/mobile/ui/activities/PrivacyPolicyActivity.kt

```diff
@@ -7,52 +7,44 @@ import android.os.Bundle
 import android.view.View
 import android.webkit.WebView
 import android.webkit.WebViewClient
-import android.widget.ProgressBar
 
-import butterknife.BindView
-import butterknife.ButterKnife
 
 import org.mifos.mobile.R
+import org.mifos.mobile.databinding.ActivityPrivacyPolicyBinding
 import org.mifos.mobile.ui.activities.base.BaseActivity
 
 /**
  * @author Rajan Maurya
  * On 11/03/19.
  */
 class PrivacyPolicyActivity : BaseActivity() {
-
-    @JvmField
-    @BindView(R.id.webView)
-    var webView: WebView? = null
-
-    @JvmField
-    @BindView(R.id.progress_bar)
-    var progressBar: ProgressBar? = null
+    private lateinit var binding : ActivityPrivacyPolicyBinding
     private var showOrHideWebViewInitialUse = "show"
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_privacy_policy)
-        ButterKnife.bind(this)
+        binding = ActivityPrivacyPolicyBinding.inflate(layoutInflater)
+        val view = binding.root
+        setContentView(view)
         setToolbarTitle(getString(R.string.privacy_policy))
         showBackButton()
```

Comment:
we need to make changes here as well. with the current changes I dont think we will be able to see the toolbar and the backbutton

Reply:
it works totally fine, have a look : 

https://github.com/openMF/mifos-mobile/assets/59947871/c3a933c4-38db-476f-a268-263a582b2e75

Reply:
Because I've overriden setContentView(view : View?) {...} with similar functionality as before, it works fine.

Reply:
check the dark mode to see if it works as expected @gururani-abhishek